current/aut-spark-submit-app.md

```diff
@@ -0,0 +1,128 @@
+# Using the Toolkit with spark-submit
+
+The Toolkit offers a variety of extraction jobs with `spark-submit`. These
+extraction jobs have a few configuration options, and analysis can use RDD or
+DataFrame is most cases.
+
+The extration jobs have a basic outline of:
+
+`spark-submit --class io.archivesunleashed.app.CommandLinAppRunner PATH_TO_AUT_JAR --extractor EXTRACTOR --input INPUT DIRECTORY --output OUTPUT DIRECTORY`
```

Comment:
is it possible to put in an example of how this script would look if one of us were to run it, below the basic outline? I find that when I have an example it's a bit easier to see what needs to be changed in line.